project/src/main/kotlin/org/ionproject/core/programme/representations/ProgrammeListRepr.kt

```diff
@@ -33,7 +33,7 @@ fun List<Programme>.programmesListRepr() = SirenBuilder()
 
 private fun Programme.buildSubentities() =
         SirenBuilder(ProgrammeReducedOutputModel(id, acronym))
-            .klass("Programme")
+            .klass("programme")
```

Comment:
This type of changes require an update to the read API.

Reply:
Will do 👍

Reply:
Opened #66